recipe/meta.yaml

```diff
@@ -7,17 +7,18 @@ package:
 source:
   fn: dask-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/d/dask/dask-{{ version }}.tar.gz
-  sha256: 4d2b0754d16ddc3f87026c1fc4fa3b589d7604a41d3f6510268f172abc1d0a5e
+  sha256: 0a6aec8b3489419c97486afd46e30b63caafc642205726a884b7fe488906d359
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  script: python -m pip install --no-deps --ignore-installed .
   noarch: python
 
 requirements:
   build:
     - python
     - setuptools
```

Comment:
Minor point: We can drop `setuptools` now as `pip` handles that. Though not a big deal either way.

Reply:
Easy enough to fix.  Thanks for shepherding me through this :)